js/index.js

```diff
@@ -8,35 +8,119 @@ const siteContent = {
     "nav-item-6": "Contact",
     "img-src": "img/logo.png"
   },
-  "cta": {
-    "h1": "DOM Is Awesome",
-    "button": "Get Started",
+  cta: {
+    h1: "DOM Is Awesome",
+    button: "Get Started",
```

Comment:
If you are going to remove the quotes around some of the keys in the object, please remove them from all of them.
Remember we want to stay consistent.

I believe prettier will do this for you automatically.